### PR TITLE
Tile Indicators: Configuration of highlight colour of the current hovered tile

### PR DESCRIPTION
**Pull request by jaysc**
Added ability to configure the highlight colour of the current hovered tile

### CODE REVIEW

**Comment by Nightfirecat** [comment]
This should probably wait until the color picker with alpha PR is merged since the current color picker doesn't allow setting the alpha.

**Comment by Nightfirecat** [comment]
Also, the default should probably stay as `new Color(0, 0, 0, 0)`

**Comment by deathbeam** [comment]
Depends on/Blocked by https://github.com/runelite/runelite/pull/6414

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	)
 	default Color highlightDestinationColor()
 	{
-		return Color.GRAY;
+		return new Color(0, 0, 0, 0);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: I think current default is fine

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 2]
```diff
@@ -63,4 +63,14 @@ default boolean highlightHoveredTile()
 	{
 		return false;
 	}
+	
+	@ConfigItem(
+		keyName = "highlightHoveredColor",
+		name = "Color of current hovered highlighting",
+		description = "Configures the highlight color of hovering tile"
+	)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: You need to add `@Alpha` annotation here for alpha picker to work

**Comment by deathbeam** [comment]
Any update on this @jaysc ?

**Comment by deathbeam** [comment]
Opened #7865

**Comment by jaysc** [comment]
> Any update on this @jaysc ?

Ah sorry, I've been away and haven't checked Github recently.

**Pull request closed without merging**